source/release-notes.txt

```diff
@@ -10,6 +10,29 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.4
+----------------
+
+New Features:
+
+-  Connect in new window with connect split-button (:issue:`COMPASS-8473`)
+
+Bug Fixes:
+
+- Update maps from map tile API v2 to raster tile API v3 (:issue:`COMPASS-8981`)
```

Comment:
Google seems to indicate that "Raster" should be capitalized.

```suggestion
- Update maps from map tile API v2 to Raster Tile API v3 (:issue:`COMPASS-8981`)
```

https://www.here.com/docs/bundle/raster-tile-api-v3-api-reference/page/index.html

Reply:
Fixed